cmd/calibrate/calibrate_test.go

```
@@ -0,0 +1,150 @@
+	os.Stderr = w
+	err := writeItemParams("unused-dsn", params, datastat.IRTSource, datastat.ScopeMeasurement, true)
+	w.Close()
+	os.Stderr = old
+	if err != nil {
+		t.Fatalf("dry-run 不应报错: %v", err)
+	}
+	io.Copy(&sb, r)
+	if !strings.Contains(sb.String(), "it01") || !strings.Contains(sb.String(), "measured_irt") {
+		t.Errorf("dry-run 输出应含 item 与 source，got: %s", sb.String())
+	}
+	if strings.Contains(sb.String(), "discrimination:1.2346") == false {
```

golangci-lint (2.12.2)

[error] SA9003: empty branch

(staticcheck)